File: 136-SingleNumber/136-SingleNumber.java
// Last updated: 14/07/2026, 14:31:40
class Solution 
{
    public int singleNumber(int[] nums) 
    {
        int n=nums.length;
        int count=0;
        for(int i=0;i<n;i++)
        {
            for(int j=0;j<n;j++)
            {
                if(i!=j)
                {
                    if(nums[i]==nums[j])
                    {
                        count++;
                    }
                }
            }
            if(count==0)
            {
                
                return nums[i];
            }
            count=0;
        }
        return -1;
    }
}

File: 136-SingleNumber/136-SingleNumber.py
# Last updated: 14/07/2026, 14:31:37
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        freq={}
        for n in nums:
            freq[n]=freq.get(n,0)+1
        for key,val in freq.items():
            if(val==1):
                return key 

        